static/js/apps/explore/success_result.tsx

```diff
@@ -53,6 +59,13 @@ import { UserMessage } from "./user_message";
 
 const PAGE_ID = "explore";
 
+const EXPERIMENT_FOLLOW_UP_ROLLOUT_RATIO = 0.2;
+
+const showFollowUpQuestions =
```

Comment:
Does this mean that for the experiment, it will be a 20% chance of seeing the follow up questions each time a query is submitted? ie a user may see both questions and topics in a single session when submitting multiple queries? (just want to confirm)